src/components/contacComponents/contactForm/ContactForm.jsx

```diff
@@ -15,7 +22,12 @@ export default function ContactForm() {
             <input type="number" name="phone" placeholder="Phone" size="10" />
           </div>
           <div className="formGroup">
-            <input type="text" name="username" placeholder="Username" size="40" />
+            <input
+              type="text"
+              name="username"
+              placeholder="Username"
```

Comment:
two inputs have a placeholder of "Username" - shouldn't this one be an input to write a message in?
If yes - it can be bigger then the other ones. You can consider <textarea> element